src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im2.contexts.onboarding.new-to-status.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.onboarding.small-option-card.view :as small-option-card]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [status-im.keycard.recovery :as keycard]
+            [status-im.multiaccounts.recover.core :as multiaccounts.recover]
+            [status-im2.common.resources :as resources]
+            [status-im2.contexts.onboarding.new-to-status.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn background-image
+  []
```

Comment:
Hi, should we use [background overlay](https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/onboarding/common/background.cljs) here?

Reply:
Hi @Parveshdhull 
I tried to use it, but had some problems:

- I modified its API to receive a specific image, since the one required for this screen is different to the added in the `background-overlay` component, the result was the image didn't look as expected, I guess it's due to its size (width and height) and resize-mode not being set as expected.

- It has a linear gradient, which is not required for this screen, I double checked in the designs and I couldn't find that linear gradient.

So finally I thought this was not the same component but they look similar.

I'll isolate this `background-image` component and the `navigation-bar` (also added in this PR) in my next PR (another screen) to make them available for all the onboarding screens since they look exactly the same.


What do you think? probably I'm wrong or missed something :thinking:  in that case, would be good if you could help me to use that suggested component :)

Reply:
Hi @ulisesmac, I talked with the design team and all screens should use the same endless scrollable illustrator. The current background overlay implementation uses illustrator from the app-locked screen, which is old and will be replaced.

https://github.com/status-im/status-mobile/issues/15012 https://github.com/status-im/status-mobile/issues/15274

I am going to address this change while fixing the issues. So you can use the background overlay screen in its current state for now. Or if you like you can keep PR implementation for now, and I can replace it once the background overlay is improved.

Reply:
> It has a linear gradient, which is not required for this screen

Thank you for pointing this, I will ask design team why other screens have that.

Reply:
>  Or if you like you can keep PR implementation for now, and I can replace it once the background overlay is improved.

@Parveshdhull OK! I'll keep this PR implementation since it looks as the designs (at least on Android), thanks for clarifying this! :)

When the background-overlay is ready you or me could do the replacement :smile: